utils.go

```diff
@@ -138,14 +139,23 @@ func resolveAddress(addr []string) string {
 	switch len(addr) {
 	case 0:
 		if port := os.Getenv("PORT"); port != "" {
-			debugPrint("Environment variable PORT=\"%s\"", port)
-			return ":" + port
+			if isValidPORTEnvVar(port) {
+				debugPrint("Environment variable PORT=\"%s\"", port)
+				return ":" + port
+			}
 		}
-		debugPrint("Environment variable PORT is undefined. Using port :8080 by default")
+		debugPrint("Environment variable PORT is undefined or invalid. Using port :8080 by default")
 		return ":8080"
 	case 1:
 		return addr[0]
 	default:
 		panic("too many parameters")
 	}
 }
+
+// Determine the PORT environment variable whether is valid。
+// If the PORT can be parsed to uint(0-65535)，return true。
+func isValidPORTEnvVar(portString string) bool {
```

Comment:
use `isValidPortEnvVar`?

Reply:
@bestgopher @thinkerou The function is no problem. The function naming can indeed be changed, such as this isValidPortEnvVar Would be better

Reply:
why use uint or uint16, not uint32?